Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollbarsController.mm

```diff
@@ -40,6 +40,12 @@
 {
 }
 
+void RemoteScrollbarsController::rtlStyleDidChange(bool shouldPlaceVerticalScrollbarOnLeft)
+{
+    if (auto scrollingCoordinator = m_coordinator.get())
```

Comment:
`auto` -> `RefPtr`. I assume it already is but we shouldn't hide it.

Reply:
Fixed.